groups/prometheus/profiles/development-eu-west-2/devops/vars

```diff
@@ -0,0 +1,4 @@
+
+instance_count = 1
```

Comment:
We need to keep these vars files consistent. We're missing a label and have `environment` in this file vs `tag_environment` in the sandbox vars file